From 44233bb2fca8284fe18207aee4c9766ed885aeca Mon Sep 17 00:00:00 2001
From: Dhaval Khandla <dhavaljk@ti.com>
Date: Tue, 28 Jan 2020 17:44:32 +0530
Subject: [PATCH 4/6] arm64: dts: ti: Increase memory window size for NTB

The default memory window size for NTB is 1MB. It is very small for a
typical application using the PCIe backplane functionality. This patch
increases the size of one memory window, so that a bigger memory chunk
can be exposed. The default choice of memory window would be first
memory window, but because it shares the BAR with other region in the
internal implementation of NTB, some errors are observed when a bigger
size is given. Hence, the second memory window is chosen.

Signed-off-by: Dhaval Khandla <dhavaljk@ti.com>
---
 arch/arm64/boot/dts/ti/k3-j721e-pcie-backplane.dtso | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/ti/k3-j721e-pcie-backplane.dtso b/arch/arm64/boot/dts/ti/k3-j721e-pcie-backplane.dtso
index 80080634b0cb..88e5190734c3 100644
--- a/arch/arm64/boot/dts/ti/k3-j721e-pcie-backplane.dtso
+++ b/arch/arm64/boot/dts/ti/k3-j721e-pcie-backplane.dtso
@@ -25,7 +25,7 @@
 				vendor-id = /bits/ 16 <0x104c>;
 				device-id = /bits/ 16 <0xb00d>;
 				num-mws = <4>;
-				mws-size = <0x100000>, <0x100000>, <0x100000>, <0x100000>;
+				mws-size = <0x100000>, <0x40000000>, <0x100000>, <0x100000>;
 			};
 		};
 	};
-- 
2.17.1

